server/src/test/java/org/opensearch/transport/RemoteClusterServiceTests.java

```diff
@@ -553,11 +553,11 @@ public void testRemoteNodeRoles() throws IOException, InterruptedException {
         final Settings settings = Settings.EMPTY;
         final List<DiscoveryNode> knownNodes = new CopyOnWriteArrayList<>();
         final Settings data = nonMasterNode();
-        final Settings dedicatedMaster = masterOnlyNode();
+        final Settings dedicatedClusterManager = masterOnlyNode();
```

Comment:
`masterOnlyNode` -> `clusterManagerOnlyNode`

Reply:
`masterOnlyNode()` is a public method, so I plan to change it in the next phase when renaming all the public Java API, and targeted for 3.0 version.
Thank you for catching these missed references.